langchain4j-core/src/main/java/dev/langchain4j/model/embedding/EmbeddingModel.java

```diff
@@ -13,18 +13,34 @@
  */
 public interface EmbeddingModel {
 
+    /**
+     * Embed a text.
+     *
+     * @param text the text to embed.
+     * @return the embedding.
+     */
     default Response<Embedding> embed(String text) {
         return embed(TextSegment.from(text));
     }
 
+    /**
+     * Embed a text segment.
```

Comment:
Probably worth mentioning that `TextSegment.metadata` is not embedded, just the `TextSegment.text`.

Reply:
done